src/models/client-sdk.ts

```diff
@@ -0,0 +1,26 @@
+import { AdditionalHttpHeaders, ClientSDKHeaderValues } from '@yext/answers-core';
+
+/**
+ * Allows users to specify additional values for specific HTTP headers.
+ *
+ * @public
+ */
+export interface HeadlessAdditionalHttpHeaders extends AdditionalHttpHeaders {
```

Comment:
Do we really need this additional `interface` here? I'm not sure how much value it really adds and it adds complexity to the code and mental model. Could we not just have the `interface` below and then do something like:
```
const values: ClientSDKHeaderValuesExcludingHeadless = blah;
const additionalHeaders: AdditionalHttpHeaders = {
     'Client-SDK': values
};
```

Reply:
I added this interface to mimic the structure in Core. `AdditionalHttpHeaders` was added in Core because we might allow more headers in the future. In that case, I assumed we would allow more headers to be specified in Headless as well (through `provideAnswersHeadless`).
If we didn't have the `HeadlessAdditionalHttpHeaders` interface, would we just accept an object of type `ClientSDKHeaderValuesExcludingHeadless` in `provideAnswersHeadless` and populate `additionalHeaders` in the function?